frontend/src/exams/exam-inventory-table.vue

```diff
@@ -239,9 +239,9 @@
                 <b-dropdown-item size="sm"
                                  v-if="!row.item.booking || Object.keys(row.item.booking).length === 0"
                                    @click="addBookingRoute(row.item)">Schedule Exam</b-dropdown-item>
-                <b-dropdown-item v-else
+                <!--<b-dropdown-item v-else
```

Comment:
🤔 Why is Update Booking commented out here?

Reply:
Double "Update Booking" dropdown item that I found in testing. Commented out instead of deleting it just in case.